src/config.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * List of rule id(s) to ignore running axe-core against
+ */
+const ignoreRulesIds = [
+  /**
+   * Rule: meta element has no refresh delay
+   * Url: https://act-rules.github.io/rules/bc659a
+   *  - This is ignored from generating a mapping in axe-core
+   */
+  'bc659a'
+]
+
+/**
+ * File extensions for which to ignore running `axe`
+ */
+const inapplicableFileExtensions = [
+  'js',
+  'xml',
+  'svg'
+]
+
+/**
+ * Specific options/ config to pass to `axe.run` per ruleId
+ */
+const rulesAxeOptions = {
+  /**
+   * Rule: role attribute has valid value
+   * Url: https://act-rules.github.io/rules/674b10
+   *  - Disable below checks
+   *    - `fallbackrole`
+   */
+  '674b10': {
```

Comment:
I want to keep this specific to per rule basis, for debugging sanity. If this becomes a common thing, then I can extract it to default options.